qf_lib/common/utils/error_handling.py

```diff
@@ -55,4 +55,20 @@ def wrap(wrapped_cls):
 
     @classmethod
     def get_error_messages(cls):
+        """
+        Method used to get the error messages
+        """
         return cls._error_messages
+
+    @classmethod
+    def reset_error_messages(cls, show_old_messages: bool = False):
+        """
+        Method used to reset the error messages to their initial value
+        @param show_old_messages:bool
```

Comment:
Please use the same doc strings notation as in other files to make it available in sphinx

Reply:
Done